### PR TITLE
Made some minor content fixes

### PR DESCRIPTION
**Pull request by sfaber34**
I took a look through the doc pages and made a handful of changes
- Standardized capitalization for titles/sub-titles
- Fixed capitalization for product names (Scaffold-Eth 2, GitHub, ...)
- Made a number of small grammar fixes

Apologies that I don't have a detailed change list here...

@austintgriffith @ZakGriffith @carletex

### CODE REVIEW

**Comment by carletex** [comment]
This is great @sfaber34. 

Left a comment/question.

Could you also check the prettier linting? The tests are failing. You can run `npm run format` and it should _fix_ the files with prettier errors.

Thank you!

File: docs/intro.md [hunk 1]
```diff
@@ -5,26 +5,26 @@ slug: /
 
 # 🏗 Welcome to Scaffold-ETH 2
 
-[Scaffold-eth](https://github.com/scaffold-eth/scaffold-eth-2) is everything you need to get started building decentralized applications on Ethereum! 🚀
+[Scaffold-Eth](https://github.com/scaffold-eth/scaffold-eth-2) is everything you need to get started building decentralized applications on Ethereum! 🚀
```
[the remark this answers is not in the document]
Comment (sfaber34) [reply docs/intro.md:8]: Yeah agreed, should be Scaffold-ETH. Didn't capitalize enough stuff!

File: docs/components/EtherInput.md [hunk 2]
```diff
@@ -24,10 +24,10 @@ const [ethAmount, setEthAmount] = useState("");
...
-| **name** (optional)        | `string`   | `undefined`   | Helps identify the data being sent if EtherInput is submitted into a form.             |
-| **disabled** (optional)    | `boolean`  | `false`       | When set to `true` changes input background color and border to have disabled styling. |
+| Prop                       | Type       | Default Value | Description                                                                             |
+| -------------------------- | ---------- | ------------- | --------------------------------------------------------------------------------------- |
+| **value**                  | `string`   | `undefined`   | You can enter ether quantity or USD quantity, but value will always be stored in ETH.   |
+| **onChange**               | `function` | `undefined`   | A callback invoked when the amount in the EtherInput changes.                           |
+| **placeholder** (optional) | `string`   | `undefined`   | The string that will be rendered before EtherInput has been entered.                    |
```
Comment (technophile-04) [inline docs/components/EtherInput.md:31]:
```suggestion
| **placeholder** (optional) | `string`   | `undefined`   | The string that will be rendered when there is not input value.                    |
```

File: docs/intro.md [hunk 3]
```diff
@@ -5,21 +5,21 @@ slug: /
...
 
-⚙️ Built using NextJS, RainbowKit, Hardhat, Foundry, Wagmi, and Typescript.
+⚙️ Built using NextJS, RainbowKit, Hardhat, Foundry, Wagmi, and TypeScript.
 
 ## About Scaffold-ETH 2
 
-Scaffold-eth is an open-source, up-to-date toolkit for building decentralized applications (dapps) on the Ethereum blockchain. It's designed to make it easier for developers to create and deploy smart contracts and build user interfaces that interact with those contracts.
+Scaffold-Eth is an open-source, up-to-date toolkit for building decentralized applications (dapps) on the Ethereum blockchain. It's designed to make it easier for developers to create and deploy smart contracts and build user interfaces that interact with those contracts.
```
Comment (technophile-04) [inline docs/intro.md:14]:
```suggestion
Scaffold-ETH is an open-source, up-to-date toolkit for building decentralized applications (dapps) on the Ethereum blockchain. It's designed to make it easier for developers to create and deploy smart contracts and build user interfaces that interact with those contracts.
```

File: docs/intro.md [hunk 4]
```diff
@@ -5,21 +5,21 @@ slug: /
...
 
 - ✅ **Contract Hot Reload**: Your frontend auto-adapts to your smart contract as you edit it.
 - 🔥 **Burner Wallet & Local Faucet**: Quickly test your application with a burner wallet and local faucet.
 - 🔐 **Integration with Wallet Providers**: Connect to different wallet providers and interact with the Ethereum network.
 
 ## Scaffold-ETH 2 Tech Stack
 
-Scaffold-eth is not a product itself but more of a combination or stack of other great tools. It allows you to quickly build and iterate over your smart contracts and frontends.
+Scaffold-Eth is not a product itself but more of a combination or stack of other great tools. It allows you to quickly build and iterate over your smart contracts and frontends.
```
Comment (technophile-04) [inline docs/intro.md:22]:
```suggestion
Scaffold-ETH is not a product itself but more of a combination or stack of other great tools. It allows you to quickly build and iterate over your smart contracts and frontends.
```

**Comment by technophile-04** [comment]
Thanks @sfaber34 this is great! 

Just added a couple of suggestions, let's fix them and I think this is ready to go 🙌

**Pull request merged**